.yaspeller.json

```
@@ -16,6 +16,10 @@
     "флексбокс(ов|ам|а|)",
     "футер",
     "хэдер",
-    "шорткат(ами|)"
+    "шорткат(ами|)",
+    "колбэк(и|ом|ах|ами|ов|)",
+    "промис(ы|а|ов|ах|ификация|ифицировать|)",
+    "брейкпоинт(ы|)",
+    "Робертса"
```

```suggestion
    "Робертс(а|у|ом|)"
```

Может сразу сделать все варианты на случай, если фамилия всплывёт ещё раз?

---

Плюс, может рассортируем по алфавиту?

---

он знает другие варианты, но не показывает именно этот.

По алфавиту по мне так не имеет смысла сортировать, мы же не будем читать этот массив

---

> он знает другие варианты, но не показывает именно этот.

Я проверил, не знает. Может всё же добавим? Только вот так:

`"Робертс(а|е|у|ом|)"`